src/app/offers/offer-details/offer-details.component.ts

```diff
@@ -139,4 +148,14 @@ export class OfferDetailsComponent implements OnInit {
     backToOffersScreen() {
         this.router.navigate(['dash/offers']);
     }
+
+    openModal() {
+        this.bsModalRef = this.modalService.show(MapModalComponent, {
+            initialState: {
+                projectLat: this.offerModel.location.lat,
+                projectLong: this.offerModel.location.long
+            }
+        });
+        this.bsModalRef.setClass('modal-lg modal-dialog-centered');
```

Comment:
Why these classes are not set inside `MapModalComponent`'s template?

Reply:
I tried to add them on new HTML element, but modal would break.

Angular dynamically adds containers where these classes should go so
they have to be defined with bsModalRef